fontforge/sfd.c

```diff
@@ -4032,13 +4032,16 @@ static void SFDGetSpiros(FILE *sfd,SplineSet *cur) {
     while ( fscanf(sfd,"%lg %lg %c", &cp.x, &cp.y, &cp.ty )==3 ) {
 	if ( cur!=NULL ) {
 	    if ( cur->spiro_cnt>=cur->spiro_max )
-		cur->spiros = realloc(cur->spiros,(cur->spiro_max+=10)*sizeof(spiro_cp));
+		cur->spiros = realloc(cur->spiros,
+		                      (cur->spiro_max+=10)*sizeof(spiro_cp));
```

Comment:
```python
>>> L1="                cur->spiros = realloc(cur->spiros,(cur->spiro_max+=10)*sizeof(spiro_cp));"
>>> L2="""
		cur->spiros = realloc(cur->spiros,
		                      (cur->spiro_max+=10)*sizeof(spiro_cp));"""
>>> import re
>>> re.sub(r'\s+', '', L2) == re.sub(r'\s+', '', L1)
True
```

What is the purpose of the added line break?

Reply:
Just increasing conformance with the mostly-followed 80 char/line rule while I was in the neighborhood.